File: Selenium/Project/Activity1.java
package HRM;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.WebDriver;
        import org.openqa.selenium.chrome.ChromeDriver;
        import org.testng.Assert;
        import org.testng.annotations.AfterTest;
        import org.testng.annotations.BeforeTest;
        import org.testng.annotations.Test;

public class Activity1{

    WebDriver driver;

    @BeforeTest
    public void setUp() {
        WebDriverManager.chromedriver().setup();
        driver = new ChromeDriver();
        // Open the URL
        driver.get("http://alchemy.hguy.co/orangehrm");
    }

    @Test
    public void verifyWebsiteTitle() {
        // Get the title of the website
        String actualTitle = driver.getTitle();

        // Expected title
        String expectedTitle = "OrangeHRM";

        // Verify the title matches exactly
        Assert.assertEquals(actualTitle, expectedTitle, "Title doesn't match");
    }

    @AfterTest
    public void tearDown() {
        driver.quit();
    }
}

File: Selenium/Project/Activity1.py
# Set up the Chrome Driver
from selenium import webdriver
from selenium.webdriver.chrome.service import Service as ChromeService
from webdriver_manager.chrome import ChromeDriverManager
driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))
#Open the URL
driver.get("http://alchemy.hguy.co/orangehrm");

# Get the title of the website
website_title = driver.title

# Expected title
expected_title = "OrangeHRM"

# Verify if the title matches
if website_title == expected_title:
    print("Website title matches.")
else:
    print(f"Website title does not match. Expected: {expected_title}, Actual: {website_title}")

# Close the browser
driver.quit()